controllers/grid/submissions/activeSubmissions/ActiveSubmissionsListGridHandler.inc.php

```diff
@@ -100,7 +100,7 @@ protected function loadData($request, $filter) {
 		}
 
 		$nonExistingUserId = 0;
-		return $submissionDao->getActiveSubmissions($context->getId(), $title, $author, $editor, $stageId, $rangeInfo);
+		return $submissionDao->getActiveSubmissions($context->getId(), $title, $author, $editor, $stageId, $rangeInfo, true);
```

Comment:
Yes, maybe, if it can be checked per default -- I think it would be good to immediately see them...